plugins/cocoapods/src/index.ts

```diff
@@ -144,34 +197,74 @@ export default class CocoapodsPlugin implements IPlugin {
       auto.prefixRelease(getVersion(this.options.podspecPath))
     );
 
-    auto.hooks.version.tapPromise(this.name, async ({ bump, dryRun, quiet }) => {
-      const previousVersion = getVersion(this.options.podspecPath);
-      const releaseVersion = inc(previousVersion, bump as ReleaseType);
+    auto.hooks.version.tapPromise(
+      this.name,
+      async ({ bump, dryRun, quiet }) => {
+        const previousVersion = getVersion(this.options.podspecPath);
+        const releaseVersion = inc(previousVersion, bump as ReleaseType);
 
-      if (dryRun && releaseVersion) {
-        if (quiet) {
-          console.log(releaseVersion);
-        } else {
-          auto.logger.log.info(`Would have published: ${releaseVersion}`);
+        if (dryRun && releaseVersion) {
+          if (quiet) {
+            console.log(releaseVersion);
+          } else {
+            auto.logger.log.info(`Would have published: ${releaseVersion}`);
+          }
+
+          return;
         }
 
-        return;
-      }
+        if (!releaseVersion) {
+          throw new Error(
+            `Could not increment previous version: ${previousVersion}`
+          );
+        }
 
-      if (!releaseVersion) {
-        throw new Error(
-          `Could not increment previous version: ${previousVersion}`
-        );
+        await updatePodspecVersion(this.options.podspecPath, releaseVersion);
+
+        await execPromise("git", [
+          "tag",
+          releaseVersion,
+          "-m",
+          `"Update version to ${releaseVersion}"`,
+        ]);
       }
+    );
 
-      await updatePodspecVersion(this.options.podspecPath, releaseVersion);
-      await execPromise("git", [
-        "tag",
-        releaseVersion,
-        "-m",
-        `"Update version to ${releaseVersion}"`,
-      ]);
-    });
+    auto.hooks.canary.tapPromise(
+      this.name,
+      async ({ bump, canaryIdentifier, dryRun, quiet }) => {
+        if (!auto.git) {
+          return;
+        }
+
+        const lastRelease = await auto.git.getLatestRelease();
+        const current = await auto.getCurrentVersion(lastRelease);
```

Comment:
@hipstersmoothie does this need to return the last canary version that was pushed? if so I'll need to commit the updated podspec, which i was originally doing, but i think it might not be necessary)

Reply:
nope